### ISSUE TITLE
/admin/donations page should have field for "mission control"

### ISSUE THREAD

**Issue by echarlie** [wuvt/wuvt-site#216]

**Comment by echarlie** [comment]
this could be a bool, or could be the name of the human who was logged in at the time

**Comment by mutantmonkey** [comment]
Yes, I think keeping track of who mission control was when a donation was made would be very useful.

**Comment by mutantmonkey** [comment]
Mission control doesn't have auth, and we've tried to implement it for years and it has never worked out. Should we just close this?
